tasks/Task.cpp

```diff
@@ -136,12 +151,13 @@ void Task::updateHook()
         m_driver->setJointCommand(command);
         auto const& messages = m_driver->queryJointCommandDownload();
         writeSDOs(messages);
-    }
 
-    TaskBase::updateHook();
+        m_input_deadline = base::Time::now() + m_input_deadline;
```

Comment:
```suggestion
        m_input_deadline = base::Time::now() + m_input_timeout;
```

Found this when testing live